src/RoboFile/ClientContainer.php

```diff
@@ -33,7 +33,11 @@
 	public function runContainerTests(
 		$opts = array(
 			'env' => 'desktop',
-			'debug' => false
+			'debug' => false,
+			'single' => false,
```

Comment:
I'm not sure if this is actually needed here, or if the `runTestSingle` function should be called directly?  and in any case the `runContainerTestPreparation` can be called independently in that case, one time per container before the actual tests start (thinking about the main coordinator for that of course)